schema/deploy/tables/project_revision_001.sql

```diff
@@ -0,0 +1,10 @@
+-- Deploy cif:tables/project_revision_001 to pg
+
+begin;
+
+alter table cif.project_revision add column revision_type varchar(1000) not null references cif.revision_type(type), add column comments varchar(10000);
```

Comment:
This will not work, given that the `project_revision` table contains committed revisions. We need to set a default value. Triggers need to be disabled and re-enabled here

Reply:
Now that there's a default value, do we even need the `add_general_revision` function?